.github/workflows/build.yml

```diff
@@ -14,10 +14,6 @@ jobs:
           node-version: '14'
           check-latest: true
       - uses: microsoft/playwright-github-action@v1
-      - uses: actions/cache@v2
```

Comment:
I tried several different things to deal with the missing playwright deps

1. setting PLAYWRIGHT_BROWSERS_PATH=0 (both on the command line and as a env variable in the workflow file)
2. adding a  `run: npx playwright install-deps` after yarn install
3. trying to find the folder that the browsers are installed in and caching that (this might actually work but I didn't find the path to the browser executables)

Since there is no `npm ci` for yarn this seems like the best way to make sure we always get the playwright browsers installed for testing.